book/website/project-design/data-governance/bigcode_casestudy.md

```diff
@@ -0,0 +1,65 @@
+(pd-dg-bigcode)=
+
+# Case Study: BigCode Data Governance
+_This case study was adapted from the [BigCode Governance card](https://huggingface.co/datasets/bigcode/governance-card) in May 2023._
```

Comment:
Could this be done as an `{admonition}` or so, to make it stand out a bit from the rest of the text?

Reply:
Thank you for reading through and the thoughtful comments, @kallewesterling!